src/kafka-consumer.cc

```diff
@@ -755,6 +767,14 @@ NAN_METHOD(KafkaConsumer::NodeAssignments) {
     Conversion::TopicPartition::ToV8Array(consumer->m_partitions));
 }
 
+NAN_METHOD(KafkaConsumer::NodeIsRebalancing) {
+  Nan::HandleScope scope;
+
+  KafkaConsumer* consumer = ObjectWrap::Unwrap<KafkaConsumer>(info.This());
+
+  info.GetReturnValue().Set(consumer->IsRebalancing());
```

Comment:
@emelyanovtv did you test this already, did it work?

I'm not sure, but maybe this line should be instead:
```suggestion
info.GetReturnValue().Set(Nan::New<v8::Boolean>(consumer->IsRebalancing()));
```

Like they have in here:
https://github.com/6fold/node-rdkafka/blob/07660be8778c0a901140504339e2338d7ba12958/src/errors.cc#L39